### PR TITLE
Add FluffyChat

### PR DESCRIPTION
**Pull request by ChristianPauly**
This should add FluffyChat to the client list. I'm not 100% that I have done everything correctly. It looks like my VS code is formatting the JavaScript different. I hope this is okay... **hides**

Signed-off-by: Christian Pauly <christian-pauly@posteo.de>

### CODE REVIEW

File: src/open/clients/Fluffychat.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        }
+    }
+
+    getDeepLink(platform, link) { }
+    canInterceptMatrixToLinks(platform) {
+        return platform === Platform.Android;
+    }
+
+    getPreferredWebInstance(link) {
+        return "fluffychat.im/web";
```
Comment (bwindels) [inline src/open/clients/Fluffychat.js:82]: If you don't support alternative instances, best to not return anything here.

**Review by bwindels — COMMENTED** [review]
Looks good, will try it out locally before merging. Thanks!

File: src/open/clients/Fluffychat.js (hunk 1 above)
Comment (bwindels) [reply src/open/clients/Fluffychat.js:82]: Yeah, just tried it out. Like it is it looks like this for web:
image

Comment (ChristianPauly) [reply src/open/clients/Fluffychat.js]: Okay I have updated it to: `getPreferredWebInstance(link) {}`
Entry looks good :-)

**Comment by bwindels** [comment]
Thanks for the changes, the PR looks good now!
Just need you to [sign off](https://github.com/matrix-org/matrix-js-sdk/blob/master/CONTRIBUTING.rst#sign-off) your commits or PR and then we can merge.

**Review by bwindels — APPROVED** [review]

**Comment by ChristianPauly** [comment]
> If you agree to this for your contribution, then all that's needed is to include the line in your commit or pull request comment:

So I have added the line to the Pull Request comment. Thats enough?

**Pull request merged**